Kernel/Bus/USB/USBPipe.cpp

```diff
@@ -137,6 +137,16 @@ InterruptInPipe::InterruptInPipe(USBController const& controller, u8 endpoint_ad
 {
 }
 
+ErrorOr<NonnullLockRefPtr<Transfer>> InterruptInPipe::interrupt_in_transfer(size_t length, u16 ms_interval, usb_async_callback callback)
```

Comment:
The name here sounds like it could be trying to interrupt an in-progress transfer.

Perhaps `create_input_transfer_from_interupt` or similar?